README.md

```diff
@@ -341,6 +341,7 @@ In your swift source declaring your custom `UIViewController` class:
 * Use the `StoryboardSceneBased` protocol if scene in your storyboard has the same `sceneIdentifier` as the name of the ViewController's class, but the `*.storyboard` file name doesn't necessary match the ViewController's class name.
   * This is typically ideal for secondary scenes in bigger storyboards
   * You'll then be required to implement the `sceneStoryboard` type property to indicate the storyboard it belongs to.
+* Use the `NibBased` protocol if your ViewController is nib-based and has set it's "File's Owner" and is the root view of the nib
```

Comment:
I don't see `protocol NibBased { …}` defined anyway in this PR?

Also, couldn't we re-use the existing `NibLoadable` protocol to apply to ViewControllers instead of creating a new protocol? (Just asking, didn't look into it to check if it actually made sense)

Reply:
Somehow the file wasn't pushed.
But since you're right that we can reuse the NibLoadable protocol, I applied this changes. Hopefully this time everything works as expected!